examples/showVisitSkyMap.py

```diff
@@ -71,6 +72,16 @@ def main(rootDir, tract, visits, ccds=None, ccdKey='ccd', showPatch=False, saveF
                     decs += dec
                     color = ('r', 'b', 'c', 'g', 'm')[i_v%5]
                     pyplot.fill(ra, dec, fill=True, alpha=0.2, color=color, edgecolor=color)
+                    if showCcds:
+                        minPoint = afwGeom.Point2D(min(ra), min(dec))
+                        maxPoint = afwGeom.Point2D(max(ra), max(dec))
+                        currentBbox = afwGeom.Box2D(minPoint, maxPoint)
```

Comment:
Ah, I see.  If I understand correctly, the previous bbox is in the pixel frame so you see same integers for all CCDs.